File: gopaal/gopaal.c
#include <stdio.h>

int main() {
    int t;  // no. of test cases
    scanf("%d", &t);

    // NOTE: this short expression is helpful in cp
    while (t--) {
        int x, y;
        scanf("%d %d", &x, &y);

        // either truck driver must turn, or Gopaal must turn...
        if (x <= 30 || y <= 20)
            printf("0\n");  // bach gyaa
        else
            printf("1\n");  // RIP Gopaal
    }
}

File: gopaal/gopaal.py
t = int(input())  # no. of test cases

for _ in range(t):
    x, y = [int(z) for z in input().split()]

    # either truck driver must turn, or Gopaal must turn...
    if x <= 30 or y <= 20:
        # bach gyaa
        print(0)
    else:
        # RIP Gopaal
        print(1)
